.github/PULL_REQUEST_TEMPLATE/HOTFIX.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Hotfix YYYY-MM-DD
```

Comment:
The example site isn't semantically versioned and changes made only to it will not result in a version bump for the library or a corresponding version tag. Using the current date seems like a Good Enough solution for distinguishing hotfixes from each other.